code/modules/mob/living/carbon/shock.dm

```diff
@@ -8,8 +8,7 @@
 	1	* src.getToxLoss() + 		\
 	1	* src.getFireLoss() + 		\
 	1	* src.getBruteLoss() + 		\
-	1	* src.getCloneLoss() + 		\
-	1	* src.halloss
+	1	* src.getCloneLoss()
```

Comment:
.... why does this use 1 * getX()

not that you need to fix it, but, ... why? did they assume this would remove floating point numbers? because it won't, all numbers in byond are floating point